sdk/go/common/resource/plugin/langruntime_plugin.go

```diff
@@ -197,19 +197,25 @@ func (h *langhost) Run(info RunInfo) (string, bool, error) {
 	for i, k := range info.ConfigSecretKeys {
 		configSecretKeys[i] = k.String()
 	}
+	configPropertyMap, err := MarshalProperties(info.ConfigPropertyMap,
+		MarshalOptions{KeepUnknowns: true, KeepSecrets: true, SkipInternalKeys: true})
```

Comment:
We should probably use RejectUnknowns rather than KeepUnknowns here. We shouldn't ever expect unknown values in config.